src/VolumeDrawable.ts

```diff
@@ -343,7 +339,10 @@ export default class VolumeDrawable {
   }
 
   setOrthoThickness(value: number): void {
-    !this.PT && this.meshVolume.setOrthoThickness(value);
+    if (this.volumeRendering instanceof PathTracedVolume) {
```

Comment:
can we prefer checking the render mode as opposed to using instanceof whenever possible.  the use of instanceof is breaking the polymorphism so should be avoided in general. when you have to use it with a polymorphic type it means something's wrong with the interface...

Reply:
That's a good way to think about it, thank you. Fixing this!